templates/events/manager/events/instance_create_update.html

```diff
@@ -66,10 +66,21 @@
                 <div class="row location-type-content physical-location-content align-items-start mb-3">
                   <div class="col-lg-6 location-search{% if ei_form.location.errors %} error{% endif %}">
```

Comment:
We could add `mb-3 mb-lg-0` here to help with spacing md-
```suggestion
                  <div class="col-lg-6 location-search mb-3 mb-lg-0{% if ei_form.location.errors %} error{% endif %}">
```